packages/react-core/src/components/Dropdown/Toggle.tsx

```diff
@@ -104,6 +104,7 @@ export class Toggle extends React.Component<ToggleProps> {
       return;
     }
     if (!this.props.bubbleEvent) {
+      this.onEscPress((event as unknown) as KeyboardEvent);
```

Comment:
can you describe the issue you were seeing

Reply:
The escape key was not closing our Dropdown component. On React 16 it was.

Reply:
have you tried the method described here?
https://reactjs.org/blog/2020/08/10/react-v17-rc.html#fixing-potential-issues
i.e. adding `{capture: true}` to the `document.addEventListener('keydown', this.onEscPress);`

Reply:
That works as well and I've updated the PR to use that instead.